server/metrics/destinations.go

```diff
@@ -0,0 +1,37 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	successDestinations *prometheus.Counter
+	errorDestination    *prometheus.Counter
+)
+
+func initDestinations() {
+	successDestinations = NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "eventnative",
+			Subsystem: "destinations",
+			Name:      "success",
```

Comment:
please clarify metrics names and corresponding methods.
init_success, init_error

That applies for sources to

Reply:
Many things may happens with destinations and sources. it is not clear success of what.
that is why i propose change naming to: init_success, init_error

Reply:
Renamed